riscv/processor.cc

```diff
@@ -609,7 +609,9 @@ void processor_t::set_csr(int which, reg_t val)
                  | MSTATUS_MPRV
                  | (supports_extension('S') ? MSTATUS_SUM : 0)
                  | MSTATUS_MXR | MSTATUS_TW | MSTATUS_TVM
-                 | MSTATUS_TSR | MSTATUS_UXL | MSTATUS_SXL |
+                 | MSTATUS_TSR
+                 | (supports_extension('U') ? MSTATUS_UXL : 0)
+                 | (supports_extension('S') ? MSTATUS_SXL : 0) |
```

Comment:
Since UXL and SXL are not actually writable in Spike (they're either hardwired to 0 or hardwired to MXL), I think we can delete these two lines altogether.